apps/src/lib/levelbuilder/lesson-editor/LessonEditor.jsx

```diff
@@ -123,11 +123,19 @@ class LessonEditor extends Component {
     })
       .done(data => {
         if (shouldCloseAfterSave) {
-          navigateToHref(
-            `/lessons/${this.state.originalLessonData.id}${
-              window.location.search
-            }`
-          );
+          if (data.hasLessonPlan) {
+            navigateToHref(
+              `/lessons/${this.state.originalLessonData.id}${
```

Comment:
please pass the lesson path down from the server using the `lesson_path` helper. this will make it a lot easier to find and change the lesson paths later.

Reply:
I will create a follow up PR with this

Reply:
https://github.com/code-dot-org/code-dot-org/pull/38745